metricproviders/prometheus/prometheus.go

```diff
@@ -140,12 +145,51 @@ func NewPrometheusProvider(api v1.API, logCtx log.Entry) *Provider {
 
 // NewPrometheusAPI generates a prometheus API from the metric configuration
 func NewPrometheusAPI(metric v1alpha1.Metric) (v1.API, error) {
+	secretKeys := []string{Address}
+	envValuesByKey := lookupKeysInEnv(secretKeys)
+	log.Infof("envValuesByKey: %v, len: %v", envValuesByKey, len(envValuesByKey))
```

Comment:
This is a high-frequency portion of code. I prefer that we don't logging this every time we construct a prometheus API client.